Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -1080,19 +1080,15 @@ namespace xr
             }
         }
 
-        void DestroyDisplayResources(std::function<void(void* texturePointer)> deletedTextureCallback = [](void*){})
+        void DestroyDisplayResources(std::function<arcana::task<void, std::exception_ptr>(void* texturePointer)> deletedTextureAsyncCallback = [](void*){ return arcana::task_from_result<std::exception_ptr>(); })
```

Comment:
Why do we need to specify the default value a second time here? Sorry can't remember.

Reply:
`DestroyDisplayResources` is called without the last parameter in the destructor.